app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+import datetime
+
+
+def sorted_friends(friend: dict) -> int:
+    return 0 if not friend.get("vaccine", False) or \
+        friend["vaccine"]["expiration_date"] < datetime.date.today() else 1
+
+
+def count_mask(persons: list) -> int:
+    return sum([1 for person in persons if not person["wearing_a_mask"]])
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    try:
```

Comment:
Declare a variable here that represents masks to buy count. Then use loop to iterate through friends. Increment masks counter in except NotWearingMaskError block. The functions above are not needed, they add extra complexity.

Reply:
@TarasSavchyn 
 Please do as Olena said

Reply:
> Declare a variable here that represents masks to buy count. Then use loop to iterate through friends. Increment masks counter in except NotWearingMaskError block. The functions above are not needed, they add extra complexity.

I removed the count function, but without the sorting function, the logic is lost